raspi-bootstrap.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Okay, cool. Thanks for the pull request. I think we can merge it once you remove this file.